inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -59,11 +60,82 @@ def from_circular_curves(cls, num_field_periods, num_coils, coil_currents,
             curve_coefficients = np.zeros((6, 2))
 
             # Set the non-zero coefficients
-            curve_coefficients[1, 0] = math.cos(toroidal_angle) * major_radius
-            curve_coefficients[1, 1] = math.cos(toroidal_angle) * minor_radius
-            curve_coefficients[3, 0] = math.sin(toroidal_angle) * major_radius
-            curve_coefficients[3, 1] = math.sin(toroidal_angle) * minor_radius
-            curve_coefficients[4, 1] = -minor_radius
+            circular_curve_coefficients(curve_coefficients, toroidal_angle,
+                                        major_radius, minor_radius)
```

Comment:
This would read better as `curve_coefficients = get_circular_curve_coefficients(toroidal_angle, major_radius, minor_radius`.